vpr/src/place/placer_context.h

```diff
@@ -113,6 +113,7 @@ struct PlacerMoveContext : public Context {
     // These vectors will grow up with the net size as it is mostly used to save coords of the net pins or net bb edges
     std::vector<int> X_coord;
```

Comment:
Could experiment with the speed of a array of structs (t_pos I think).

Reply:
OK, scratch that as you have to sort them.  But comment that (reason why we use independent vectors).